### PR TITLE
Rearranging navigation bar (#151)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR makes changes to `index.html` to modify the navigation bar as requested by @mfldavidson. In general, it makes the search bar more prominent while simplifying the other tabs and options. Some additional details on the changes are provided below. The PR aims to resolve issue #151.

1) I changed the link for the Student Explorer logo to direct to the "About" page, which will serve as a quasi-landing page for the time being. Note that the root route (`/`) route directs the current user to their "My Students" tab, or the `All Advisors` page if the advisor does not have a record in the `DM_MNTR` table. We will be addressing this behavior as part of a separate issue.

2) In order to make the search bar more prominent, I moved the `form` element up and modified an existing class to be `navbar-left`, resulting in the field appearing directly to the right of the Student Explorer logo.

3) As requested, the "Advisors" link was moved to the drop-down menu above "Manage". Both "Advisors" and "Manage" links are only visible to superusers. The `advisors/` route and advisor-specific routes (e.g. `advisor/ssciolla`) are still accessible directly for authenticated users. I also added some conditional handling so that both the "Advisors" and "Manage" links are highlighted in the drop-down menu when the user is on those pages.

4) The remaining links in the navigation bar ("My Students", "About", and "Resources & Support") were all moved to the right by grouping them with the drop-down menu in a `ul` with the`navbar-right` `class`.

5) I added an `if` template block to the presentation of a user's name in the drop-down menu so that it is only included when the user has a `first_name` and `last_name` in the database. This prevents some unneeded white space when the user does not have one of those values (which is the case for "admin").

6. The `url` declarations used for active link highlighting on the bar were grouped together at the beginning of the content nested under the parent `nav` element. White space was also removed from some elements to improve the file's overall readability.

### CODE REVIEW

File: student_explorer/templates/student_explorer/index.html [hunk 1]
```diff
@@ -38,6 +38,10 @@
 
     <body>
         <nav class="navbar navbar-inverse navbar-fixed-top">
+            {% url 'seumich:advisor' advisor.username as mystudentsurl%}
```
Comment (jonespm) [inline student_explorer/templates/student_explorer/index.html]:
```suggestion
            {% url 'seumich:advisor' advisor.username as mystudentsurl %}
```

**Comment by mfldavidson** [comment]
Here's a slightly out of date one Sam sent me the other day. Not My Students is moved over to the right. I saw it on Sam's computer.
image

**Comment by ssciolla** [comment]
Here's a photo of what I have now:

Screen Shot 2019-10-01 at 2 37 54 PM

File: student_explorer/templates/student_explorer/index.html (hunk 1 above)
Comment (ssciolla) [reply student_explorer/templates/student_explorer/index.html]: Done. See latest commit.

**Pull request merged**